packages/@webex/internal-plugin-ai-assistant/src/ai-assistant.ts

```
@@ -0,0 +1,447 @@
+        timer.reset();
+        const resultData = get(data, dataPath, []);
+        const errorMessage = get(data, 'response.errorMessage');
+        const errorCode = get(data, 'response.errorCode');
+
+        if (data.finished) {
+          // For finished messages, decrypt and emit the final complete message
+          timer.cancel();
+
+          try {
+            let decryptedMessage;
+            if (resultData && resultData.value) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)